org.mwc.cmap.plotViewer/src/org/mwc/cmap/plotViewer/editors/CorePlotEditor.java

```diff
@@ -553,14 +553,17 @@ protected String getFixedFilePath(final String fileName) {
 			}
 			else if(input instanceof FileStoreEditorInput) {
 				//if the file is dragged from outside workspace, get the location of the plot file 
+				String tmpFilePath;
 				try {
 					File localFile = new File(((FileStoreEditorInput)input).getURI().toURL().getFile());
-					if(localFile!=null) {

```

Comment:
`localFile` can't be null - it must contain a File object (or exception will have been thrown)